go-client/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	sw "github.com/danmanor/WebServerProject/go-client/go"
+)
+
+func main() {
+	for true {
+
+		var proceed string
+
+		fmt.Println("Do You Wish To Register A User (1), Delete A User (2), Get All Users (3) or Exit (4)?  (1/2/3/4)")
+		fmt.Scanln(&proceed)
+		switch proceed {
+		case "4":
+			break
+		case "1":
+			var name string
+			var email string
+
+			fmt.Println("Please Enter Your Name")
+			fmt.Scanln(&name)
+			fmt.Println("Please Enter Your email")
+			fmt.Scanln(&email)
+
+			cfg := sw.NewConfiguration()
+			apiService := sw.NewAPIClient(cfg)
+
+			res, _, err := apiService.DefaultApi.AddUser(nil, name, email)
+			if err != nil {
+				log.Fatal(err)
+			} else if res == "" {
+				fmt.Println("This user already exists!")
+			} else {
+				fmt.Println("Success! Your User Id Is: ", res)
+			}
+
+		case "2":
+			var userId string
+			fmt.Println("Please Enter the user's Id you want to delete")
+			fmt.Scanln(&userId)
+
+			cfg := sw.NewConfiguration()
+			apiService := sw.NewAPIClient(cfg)
+
+			_, err := apiService.DefaultApi.DeleteUser(nil, userId)
+			if err != nil {
+				fmt.Println("No Such User Exists")
+			} else {
+				fmt.Println("Success! The User Was Deleted")
+			}
+		case "3":
+			cfg := sw.NewConfiguration()
+			apiService := sw.NewAPIClient(cfg)
+
+			resp, _, err := apiService.DefaultApi.FindUsers(nil)
+			if err != nil {
+				log.Fatal(err)
+			} else {
+				for _, v := range resp {
+					fmt.Println(v.UserId, v.Name, v.Email)
+				}
+			}
+		}
```

Comment:
It is better to split it into function rather leave it all in single one:

e.g. 
```suggestion
		case "1":
                  HandleCase1()
		case "2":
                  HandleCase2()		
		case "3":
                  HandleCase3()
                
                ...  
```